models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_kyberswap_pools.sql

```diff
@@ -0,0 +1,277 @@
+{{ config(
+    schema='lido_liquidity_arbitrum',
+    alias = 'kyberswap_pools',
+    partition_by = ['time'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool', 'time'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "lido_liquidity",
+                                \'["ppclunghe", "gregshestakovlido"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2022-02-11' %} 
+
+with dates as (
+select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 hour)) as hour
+)
+ 
+, pools as (
+select pool as address, 'arbitrum' as blockchain, 'kyberswap' as project, swapFeeUnits/1000 as fee
+from {{ source('kyber_arbitrum', 'Elastic_Factory_evt_PoolCreated') }}
+where token0 = lower('0x5979D7b546E38E414F7E9822514be443A4800529') or token1 = lower('0x5979D7b546E38E414F7E9822514be443A4800529')
+
+)
+
+, tokens_mapping as (
+select * from (values 
```

Comment:
Maybe we can write `select *` in full column names?
```suggestion
select address_l1, address_l2 from (values 
```

Reply:
fixed